docs/conf.py

```diff
@@ -413,13 +413,13 @@
 
 
 def setup(sphinx):
-    sphinx.add_stylesheet("css/custom.css")
+    sphinx.add_css_file("css/custom.css")
 
     try:
         utils_path = os.path.abspath(os.path.join(__file__, "..", "utils"))
         sys.path.insert(0, utils_path)
         from StoryLexer import StoryLexer
 
-        sphinx.add_lexer("story", StoryLexer())
+        sphinx.add_lexer("story", StoryLexer)
```

Comment:
Are these changes linked to the others? From what I understand they are documentation-related

Reply:
Yes, can also remove them. The documentation built was failing on GH actions and it reported that those function are deprecated and that we should switch to xyz. That's why I updated them. But I guess it was just a random failure of GH actions and I should be able to undo those changes. Might be better to update the deprecated functions in a separate PR.